website/docs/r/ecr_repository.html.markdown

```diff
@@ -37,6 +37,13 @@ The following attributes are exported:
 * `registry_id` - The registry ID where the repository was created.
 * `repository_url` - The URL of the repository (in the form `aws_account_id.dkr.ecr.region.amazonaws.com/repositoryName`
 
+## Timeouts
+
+`aws_ecr_repository` provides the following [Timeouts](/docs/configuration/resources.html#timeouts)
+configuration options:
+
+- `read` - (Default `10 minutes`) How long to wait for a repository to be listed.
```

Comment:
This should probably specifically mention `on creation`. We also generally prefer to default to 1 minute unless there is some known documentation about eventual consistency taking a certain amount of time.